3581/3/app/helpers/user_helper.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+
+module UserHelper
```

Comment:
the whole module looks weird and unnecessary. You can do it via default active model validators or write your own custom